index.js

```diff
@@ -375,6 +375,144 @@ BabelClient.prototype.createAnnotation = function(token, data, options, callback
 };
 
 /**
+ * Update an annotation
+ *
+ * @param {string} token Persona token
+ * @param {object} data Data that can be passed into an annotation
+ * @param {object} data._id
```

Comment:
I am unsure if this is the correct notation to describe the `data` parameter. I though it was custom to drop a line then tab under the original definition to add more information?

```
@param {object} data Data that can be passed into an annotation
    Also includes the following attributes
    data._id
    data.hasBody
....
```